### PR TITLE
ft : facility registry integration

### PR DESCRIPTION
**Pull request by jabahum**
- facility registry integration

### CODE REVIEW

File: packages/esm-ugandaemr-app/src/views/home/header/ugemr-home-header.component.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  updateSelectedQueueLocationUuid,
+  updateSelectedServiceName,
+  useSelectedFacilityName,
+  useSelectedQueueLocationName,
+} from '../../../../../esm-patient-queues-app/src/helpers/helpers';
+import PatientQueueIllustration from '../../../../../esm-patient-queues-app/src/patient-queue-header/patient-queue-illustration.component';
+import { useQueueLocations } from '../../../../../esm-patient-queues-app/src/patient-search/hooks/useQueueLocations';
+import styles from './patient-home-header.scss';
+
+const PatientHomeHeader: React.FC<{ title?: string }> = ({ title }) => {
```
Comment (pirupius) [inline packages/esm-ugandaemr-app/src/views/home/header/ugemr-home-header.component.tsx:17]: Typically it's a good practice to have the component name similar to the file unless there many exported components within the same component.

**Comment by pirupius** [comment]
@jabahum please add some screenshots or a screen recording showing the work done for a quicker review

**Comment by jabahum** [comment]
@pirupius below is implementation of feature
image

**Comment by pirupius** [comment]
@jabahum did u attach it to the Home Screen?

**Comment by jabahum** [comment]
> @jabahum did u attach it to the Home Screen?

yes its on the home screen

**Review by pirupius — APPROVED** [review]

**Pull request merged**